public/locales/es/common.json

```diff
@@ -1,6 +1,8 @@
 {
     "close": "Cierra",
+    "submit": "Enviar",
```

Comment:
Verb format conjugated?

Reply:
The internet says "Enviar" is the conventional form, here's one such [reference of UX Spanish translations](https://ux.stackexchange.com/questions/80833/reference-for-spanish-ui)